### PR TITLE
OpenMP threadprivate + PTA null ptr deref fix

### PR DESCRIPTION
**Pull request by addison-coderrect**
Addresses #93.

### CODE REVIEW

**Comment by addison-coderrect** [comment]
Includes a fix in PTA for null pointer dereference due to incorrect check (getFunction => getCalledFunction).

**Comment by addison-coderrect** [comment]
Depends on #120.

File: tests/data/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+Testing/
```
Comment (funemy) [inline tests/data/.gitignore:1]: Is this a folder necessary to ignore?
If it's just for your own testing, I suggest adding this to your `.git/info/exclude` file. So it won't get pushed.

**Review by funemy — COMMENTED** [review]
LGTM overall, just a very minor suggestion.

File: tests/data/.gitignore (hunk 1 above)
Comment (addison-coderrect) [reply tests/data/.gitignore:1]: This was generated by CTest; don't think it's necessary to add, so I'll drop it.

**Pull request merged**